yoga/node/Node.h

```diff
@@ -298,6 +438,7 @@ class YG_EXPORT Node : public ::YGNode {
   Style style_;
   LayoutResults layout_;
   size_t lineIndex_ = 0;
+  size_t contentsChildren_ = 0;
```

Comment:
nit: this reads as a list of children
```suggestion
  size_t contentsChildrenCount_ = 0;
```

Reply:
Done in https://github.com/facebook/yoga/pull/1726/commits/6b6acde221894d38d5956f46f461fb2633c31915